data/coalition/coalition missions.txt

```diff
@@ -212,22 +216,22 @@ mission "Coalition: Lost: Offer"
 			`The Heliarchs are waiting for you when you land. You see no civilians as you look around, but there are enough armed guards to man several of their Punishers. Said Punishers are all parked near your own ship.`
 			`	The Saryd starts speaking, "As to be trusted, you have been deemed, provide you with a jump drive, so that stranded you are not, we will."`
 			`	"Mind you, not a gift this is, but a loan, Captain," the Kimek continues. "Another one to replace this one, you must bring us, so that losing a copy of such valuable equipment, we are not."`
-			`	"By <date>, brought here the replacement jump drive must be. If any longer you take, assume that betrayed our trust you have, we will, and an enemy, considered you will be," the Arach finishes explaining. "Agree with those terms, do you, Captain?"`
+			`	"By <date>, brought here the replacement jump drive from the Quarg must be. If any longer you take, assume that betrayed our trust you have, we will, and an enemy, considered you will be," the Arach finishes explaining. "Agree with those terms, do you, Captain?"`
 			choice
-				`	"Yes, I promise I'll return with a second jump drive."`
+				`	"Yes, I promise I'll return with a second jump drive from the Quarg.."`
 				`	"Sorry, I need more time to think on this."`
 					defer
-			`	Their engineers proceed to install the jump drive in your ship. A couple of the guards come a bit closer during the process. When it's all done, you're reminded that you need to bring another jump drive here by <date>.`
+			`	Their engineers proceed to install the jump drive in your ship. A couple of the guards come a bit closer during the process. When it's all done, you're reminded that you need to bring another jump drive from the Quarg here by <date>.`
 				accept
 	on accept
-		outfit "Jump Drive" 1
+		outfit "Quarg Jump Drive" 1
 		fail "Coalition: Lost: Marker"
 	on fail
 		"reputation: Heliarch" = -1000
 		"reputation: Coalition" = -1000
 	on complete
-		require "Jump Drive" 2
-		outfit "Jump Drive" -1
+		require "Quarg Jump Drive" 2
```

Comment:
In this case, the drive you return to the Heliarchs should be the one they gave you (I.e. the Quarg one), but there's no reason the other one should be - so you should use a `to complete` with an `or` so that either one non-Quarg JD (as well as the Quarg one) or two Quarg JDs can be used to complete the mission.

Reply:
I think I changed that, thanks!